README.md

```diff
@@ -55,6 +55,9 @@ A service that:
 - [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads) (NVIDIA Driver >= `535`, CUDA >= `12.2`)
 - [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html)
 
+> [!Note]
+> You should not need to install Python separately, but if you do, NVIDIA-Ingest only supports [Python version 3.10](https://www.python.org/downloads/release/python-3100/).
```

Comment:
Can we just say

"Python version 3.10"

We make the recommendation for a conda created separate env in step 2.